### ISSUE TITLE
Expo app stuck in splash screen

### ISSUE THREAD

**Issue by Onibenjo** [expo/expo#10738]
## 🐛 Bug Report
I initialised two expo apps, bare and managed and they both got stuck in the splash screen
### Summary of Issue 
I created an expo managed workflow app with typescript and it is stuck in the splash screen. I even updated expo-cli to 3.28.1, same problem.
I thought it was because of my node version (14.x), but i installed 12.8 and still same error

### Environment - output of `expo diagnostics` & the platform(s) you're targeting
  Expo CLI 3.28.1 environment info:
    System:
      OS: Windows 10 10.0.19041
    Binaries:
      Node: 12.18.4 - C:\Program Files\nodejs\node.EXE
      Yarn: 1.22.5 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
      npm: 6.14.6 - C:\Program Files\nodejs\npm.CMD
    SDKs:
      Android SDK:
        API Levels: 29, 30
        Build Tools: 27.0.3, 28.0.3, 29.0.2, 30.0.0, 30.0.0, 30.0.0
    IDEs:
      Android Studio: Version  3.5.0.0 AI-191.8026.42.35.6010548
    npmPackages:
      expo: ~39.0.2 => 39.0.3
      react: 16.13.1 => 16.13.1
      react-dom: 16.13.1 => 16.13.1
      react-native: https://github.com/expo/react-native/archive/sdk-39.0.3.tar.gz => 0.63.2
      react-native-web: ~0.13.12 => 0.13.18
    Expo Workflow: managed

### Steps to Reproduce
expo init

### Expected Behavior vs Actual Behavior
Load app vs stuck on splash screen

**Comment by byCedric** [comment]
Hi @Onibenjo! I'm sorry you are having issues with Expo and Windows. I can't seem to reproduce this issue on my end, could you commit the project that you get with `expo init` and push it to a repo? Maybe I can find something in there. Besides that, I think you have two additional things you can try:

1. Try running powershell with admin privileges. We have quite some issues related to Windows and permissions that might be affecting you here as well.
2. If you have WSL installed, try running this from WSL. It's practically Linux and I know we have a lot fewer issues with Linux.

**Comment by Onibenjo** [comment]
I had to uninstall and install my expo client

**Comment by Onibenjo** [comment]
Apparently, my expo app were not loading on my lolipop device (android v5.1) but it loaded on android versions higher than that and I don't understand why because it was working perfectly the day before the issue and I did not update the expo client app

**Comment by Slapbox** [comment]
I just hit this issue and had to uninstall the Expo client. It popped up while I was in the middle of making changes, so I rolled back each one with `git checkout HEAD -- [my file]` and was surprised to find that it still didn't work.

So I stopped Expo-CLI, ran the following command to clear haste/metro caches in a Cygwin Windows 7 environment:

`cd "\Users\MyUser\AppData\Local\Temp" ; rm -rf ./haste* ; rm -rf ./metro-cache* ; cd -`

then I ran `expo start -c`

I force stopped the Expo client on my Android 10 device, cleared the cache, and tried to open the app on my phone. The issue persisted.

Uninstalled Expo app and re-installed - resolved. The Javascript bundle would not even begin to build before I uninstalled.

**Comment by Onibenjo** [comment]
Alright, will try that out, thanks

**Comment by Onibenjo** [comment]
still same

**Comment by Onibenjo** [comment]
updated expo cli and it was fixed

**Comment by Slapbox** [comment]
FWIW our issue occurred on the latest CLI (`3.28.2`)

**Comment by muathime** [comment]
I had the same problem so I did code review on my code on github and found out that I was importing `useState` wrongly. Seems to have slipped on an auto import event.

`import { useState } from 'react/cjs/react.development';`
So I corrected to
`import { useState } from "react";` 

In your case it might be different but I propose searching for any import from `....react.development` and correct that.

I resolved it this way and now it works great!!!
